python/python/tests/test_async_client.py

```diff
@@ -1063,6 +1063,144 @@ async def test_linsert(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.linsert(key2, InsertPosition.AFTER, "p", "e")
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lmove(self, redis_client: TRedisClient):
```

Comment:
please add tests to `test_multi_key_command_returns_cross_slot_error` to ensure that `source` and `destination` are located in the same hash slot.